### PR TITLE
Several fixes to documentation on features released on 5.192

### PR DESCRIPTION
**Pull request by fturizo**
These fixes include expanding and clarifying the extent of customer requested features like:

* Certificate Realms Extensions
* Auditing Service
* MP OpenTracing JAX-WS Support

Also fixed some confusion on the templating feature of MP Metrics custom vendor metrics as well. Once this is approved, I'll request a follow-up PR to merge this on the `5.192` branch

### CODE REVIEW

File: documentation/payara-server/admin-console/auditing-service.adoc [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+|Description
+|Default
+|Mandatory
+
+| `--notifier`
+| String
+| The notifier to configure. One of (case insensitive):
+
+* `LOG`
+* `HIPCHAT`
```
Comment (rdebusscher) [inline documentation/payara-server/admin-console/auditing-service.adoc]: I would remove this as the service is no longer available at Atlassian.

Comment (fturizo) [reply documentation/payara-server/admin-console/auditing-service.adoc]: The Hipchat Notifier is still a documented and active feature so we can leave it there for the moment. I'll raise an issue about removing the feature and documentation.

File: documentation/payara-server/admin-console/auditing-service.adoc [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+[source, shell]
+----
+asadmin> get-admin-audit-configuration
+
+Enabled  Audit Level  
+false    MODIFIERS
+Name     Notifier Enabled
+LOG      false
+
+Command get-requesttracing-configuration executed successfully.
```
Comment (OndroMih) [inline documentation/payara-server/admin-console/auditing-service.adoc]: This should read "get-admin-audit-configuration executed successfully"

File: documentation/payara-server/server-configuration/security/security-realms.adoc [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+=== Using an `asadmin` command
+
+You can also use the following asadmin command to set the value of the property through:
+
+[source, shell]
+----
+asadmin> set configs.config.${YOUR_INSTANCE_CONFIG}.security-service.auth-realm.certificate.property.common-name-as-principal-name=true
+----
+
+IMPORTANT: After setting the value of the property, make sure that you **restart the server instance** to for the changes to take effect.
```
Comment (OndroMih) [inline documentation/payara-server/server-configuration/security/security-realms.adoc]: Remove "to" in "to for the changes to take effect"

File: documentation/payara-server/server-configuration/security/security-realms.adoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+[[using-cn-as-subject-in-applications]]
+=== Using `CN` as Subject in Applications
+
+With the feature enable, users can be correctly authenticated when a valid certificate uses the common name instead of the full domain name. It is also possible to map users to roles and groups in this manner.
```
Comment (OndroMih) [inline documentation/payara-server/server-configuration/security/security-realms.adoc]: With the feature "enabled"

File: documentation/payara-server/server-configuration/security/security-realms.adoc [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+For example if previously an user authorization roles and groups were mapped like this in the `payara-web.xml` deployment descriptor:
```
Comment (OndroMih) [inline documentation/payara-server/server-configuration/security/security-realms.adoc]: Grammar:
* For example, if user authorization roles and groups were previously mapped like this in the `payara-web.xml` deployment descriptor

File: documentation/microprofile/metrics.adoc [hunk 6]
```diff
@@ -118,7 +113,27 @@ vendor:jdbc_connection_pool_resources/_derby_pool_pool_numconnfree 0
...
 
-Here's another example of defining a custom metric definition template with multiple placeholder that will generate metrics for the number of free connections available to all JDBC connections pools:
+[[advanced-templating]]
+==== Advanced Templating
+
+_Since Payara Server 4.1.2.184 and 5.184_
+
+If you need more control on the attributes of specific MBeans used in a metric definition, you can use the `${key}`, `${attribute}` and `${subattribute}` placeholders to define more fine-grained metrics.
+
+These placeholders are used as follow:
```
Comment (OndroMih) [inline documentation/microprofile/metrics.adoc]: as follows:

File: documentation/payara-server/admin-console/auditing-service.adoc (hunk 1 above)
Comment (OndroMih) [reply documentation/payara-server/admin-console/auditing-service.adoc]: I agree to leave it for now until the feature is removed.

**Pull request merged**